dist/latest/latest.html

```diff
@@ -0,0 +1,81 @@
+<html>
```

Comment:
Might not be a good idea to put this in `dist` since it would complicate any cleaning scripts if we used them to clear out build directories. Not a problem right now but yeah if it's not hard I think it would be better to move it